source/administration/mmctl-cli-tool.rst

```diff
@@ -2355,36 +2356,65 @@ mmctl permissions add
    --local                       allows communicating with the server through a unix socket
    --strict                      will only run commands if the mmctl version matches the server one
 
-mmctl permissions role assign
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+mmctl permissions remove
+^^^^^^^^^^^^^^^^^^^^^^^^
 
 **Description**
 
-Assign users to a role by username (only available in Enterprise Edition E10 and E20).
+  Remove one or more permissions from an existing role (only available in E10 and E20).
```

Comment:
Wondering if the word "only" should be omitted.  It can have a negative connotation.

Reply:
Great catch, @wiersgallak! I've removed the word "only" and ensured that there aren't other similar instances.